frontend/resourceadm/components/AccessListMembers/AccessListMembers.tsx

```diff
@@ -118,12 +113,7 @@ export const AccessListMembers = ({
             disableButtonFn={(disableItem: AccessListMember) =>
               !!listItems.find((listItem) => disableItem.orgNr === listItem.orgNr)
             }
-            buttonNode={
-              <>
-                {t('resourceadm.listadmin_add_to_list')}
-                <PlusCircleIcon className={classes.buttonIcon} />
-              </>
-            }
+            isAdd={true}
```

Comment:
I believe this can be written as this instead to shorten it
```suggestion
            isAdd
```